### PR TITLE
Record book generation duration

### PR DESCRIPTION
**Pull request by samwilson**
Add a new 'duration' column to the DB table, to record the number
of seconds each book generation took.

Bug: T267079

### CODE REVIEW

File: src/Entity/GeneratedBook.php [hunk 1]
```diff
@@ -128,4 +139,18 @@ public function getFormat(): string {
...
+
+	/**
+	 * Get the generation duration in seconds.
+	 * @return int|null
+	 */
+	public function getDuration(): ?int {
+		return $this->duration;
+	}
+
+	public function setDuration( ?int $duration ): self {
```
Comment (MusikAnimal) [inline src/Entity/GeneratedBook.php:151]: I'm guessing you used the Symfony CLI to create these methods when you added the column. Weird how it had it return `self` when it didn't for me! Not a big deal, but I would just make it `void` to be consistent with the other methods.

Also missing the doc block.

File: src/Entity/GeneratedBook.php [hunk 2]
```diff
@@ -54,16 +55,26 @@ class GeneratedBook {
 	 */
 	private $format;
 
+	/**
+	 * @ORM\Column(type="integer", nullable=true)
+	 */
```
Comment (MusikAnimal) [inline src/Entity/GeneratedBook.php:60]: Suggest adding `@var StopwatchEvent|null $duration`

**Review by MusikAnimal — COMMENTED** [review]
Looks good! Just a few very nitpicky comments.

**Comment by MusikAnimal** [comment]
Oh, sorry didn't notice this was still WIP :)

File: src/Entity/GeneratedBook.php (hunk 2 above)
Comment (samwilson) [reply src/Entity/GeneratedBook.php:60]: Done.

File: src/Entity/GeneratedBook.php (hunk 1 above)
Comment (samwilson) [reply src/Entity/GeneratedBook.php:151]: Oh yeah, that is strange. Fixed both things.

**Review by samwilson — COMMENTED** [review]
No worries, thanks for the speedy review!

**Pull request merged**